src/Core/Rendering/RenderController.cpp

```diff
@@ -396,6 +397,8 @@ namespace MxEngine
 
         this->ComputeBloomEffect(camera, camera.HDRTexture);
         this->ApplySSGI(camera, camera.HDRTexture, camera.SwapTexture1, camera.SwapTexture2);
```

Comment:
hm... I encorage you to try testing already existing algorithms in half / quater resolution + maybe increased sample count. Smth like SSGI can really benefit from that. But maybe in a separate PR afterwards